fashion-messaging-app-api/server.js

```diff
@@ -12,6 +12,7 @@ import { sequelize } from './database.js';
 import { User, Post, SearchResult } from './models/index.js';
 import userRoutes from './routes/users.js';
 import SequelizeStoreInit from 'connect-session-sequelize';
+import { Op } from 'sequelize';
```

Comment:
The "Op" from Sequelize helps kms use logical operators for complex queries which in this case is used to query my database